contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -105,6 +105,33 @@ While some exceptions may happen, approvers should use the following guidance:
 
 [API changes]: https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md#what-parts-of-a-pr-are-api-changes
 
+### Compatibility versions
+
+The Kubernetes "compatibility version" feature promises that control-plane
+components can be configured to behave as if they were any of the 3 previous
+releases, which includes feature gates and gate-controlled APIs and logic.
```

Comment:
But I think we make an exception for alpha feature gates. Have I got that right?

Reply:
> But I think we make an exception for alpha feature gates. Have I got that right?

Yes, alpha gates are excepted.

Reply:
added a note for non-alpha